static/js/moj-filter-button-toggle.js

```diff
@@ -0,0 +1,17 @@
+new MOJFrontend.FilterToggleButton({
+  bigModeMediaQuery: '(min-width: 48.063em)',
+  startHidden: false,
+  toggleButton: {
+    container: $('.moj-action-bar__filter'),
+    showText: 'Show filter',
+    hideText: 'Hide filter',
+    classes: 'govuk-button--secondary',
+  },
+  closeButton: {
+    container: $('.moj-filter'),
+    text: 'Close',
+  },
+  filter: {
+    container: $('.moj-filter-layout__filter'),
+  },
+})
```

Comment:
basically the same as court register except by default filter is open, which makes sense since you would always want to filter I guess